src/content.ts

```diff
@@ -1,7 +1,17 @@
 import { getCollection } from 'astro:content';
 import { isEnglishEntry, isRecipeEntry, isTutorialEntry } from './content/config';
 
-export const allPages = await getCollection('docs');
+export const allPages = await getAllPages();
```

Comment:
What do you think of doing this instead of the extra function?

```suggestion
export const allPages = await getCollection('docs', (doc) => (
	import.meta.env.PUBLIC_ONLY_EN ? isEnglishEntry(doc) : true
));
```

Reply:
Ah that looks a lot better!